test/rules/no-setup-in-describe.js

```diff
@@ -190,7 +190,7 @@ ruleTester.run('no-setup-in-describe', rule, {
                 line: 1,
                 column: 28
             }, {
-                messagee: memberExpressionError,
+                message: memberExpressionError,
```

Comment:
Nice catch!

Reply:
Thanks to ESLint7! :)  (It reports such unknown properties in the tester now)